app/views/govuk_component/related_items.raw.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  sections ||= []
+%>
+<aside class="govuk-related-items" role="complementary">
+  <% sections.each do |section| %>
+    <% section_title = section[:title] || "Elsewhere on GOV.UK" %>
```

Comment:
If `Elsewhere on GOV.UK` gets passed in as a section title, then this part of the component (which was ported) reads poorly:
```
<a href="<%= section[:url] %>">
  More<% if section[:title] %> <span class="visuallyhidden">in <%= section_title %></span><% end %>
</a>
```